css/style.css

```diff
@@ -1,4 +1,572 @@
+  display: -ms-flexbox;
+  display: flex;
+}
+
+.test__right {
+  padding: 0;
+}
+
+.test__column_testdrive {
+  padding: 10px 0 0 0;
+}
+
+.test__column_testdrive.test__img {
+  margin: 2px 2px 0 0;
+}
+
+.test__column_testdrive.link_test__item {
+  text-decoration: none;
+  color: #000;
+}
+
+.test__body {
+  font-weight: 600;
+  font-size: 35px;
+  margin: 0;
+}
+
+.test__body_small {
+  font-size: 20px;
+}
+
+.test__body_bottom {
+  font-size: 12px;
+  line-height: 15px;
+  margin: 1px 0 0 0;
+}
+
+.test__body_testdrive {
+  font-weight: bold;
+  font-size: 13px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  padding: 7px 0 0 7px;
+}
+
+.test__body_testdrive:hover {
+  color: #e51937;
+  -webkit-transition: ease-in-out .5s;
+  -o-transition: ease-in-out .5s;
+  transition: ease-in-out .5s;
+  }
+
+.btn {
+  width: 160px;
+  height: 40px;
+  border: 2px solid #e51937;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  font-weight: bold;
+  font-size: 13px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #e51937;
+  padding: 10px 0 0 0;
+}
+
+.btn__test {
+  color: #e51937;
+  border: 2px solid #e51937;
+}
+
+.btn__test:hover {
+  background-color: #e51937;
+  color: #fff;
+  -webkit-transition: ease-in-out .5s;
+  -o-transition: ease-in-out .5s;
+  transition: ease-in-out .5s;
+}
+
+.button {
+  margin: 5px 0 0 0;
+}
+
+.test__img {
+  display: block;
+  margin: 5px  auto 0;
+  width: 100%;
+  height: 100%;
+  -ms-flex-item-align: start;
+  align-self: flex-start;
+}
+
+.test__arrow {
+  margin: 39px auto 0;
+  width: 30px;
+}
+
+@media (max-width: 1100px) {
+  .banner__img_background {
+    width: 1000px;
+    height: 300px;
+  } 
+}
+
+@media (max-width: 768px) {
+  .banner__img_background {
+    width: 700px;
+    height: 270px; 
+  }
+
+  .banner__img_red {
+    width: 673px;
+    height: 328px; 
+  }
+
+  .header__container {
+    max-width: 720px; 
+  }
+
+  .menu__item {
+    display: none;
+  }
+
+  .test__row {
+    margin: 0 auto;
+    -ms-flex-wrap: wrap;
+    flex-wrap: wrap;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    max-width: 490px;
+    margin: 0 auto;
+    padding: 0;
+  }
+
+  .test__row_right {
+    -webkit-box-flex: 0;
+    -ms-flex: 0 1 100%;
+    flex: 0 1 100%;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    width: 488px;
+    padding: 0;
+  }
+
+  .test__row_left {
+    -webkit-box-flex: 0;
+    -ms-flex: 0 1 100%;
+    flex: 0 1 100%;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    padding: 0;
+    margin: 55px auto 0;
+    max-width: 418px; 
+  }
+
+  .test__column {
+    -webkit-box-flex: 0;
+    -ms-flex: 0 1 50%;
+    flex: 0 1 50%;
+    margin: 0; 
+  }
+
+  .test__body {
+    font-size: 50px;
+    color: red;
+  }
+
+  .btn__test {
+    width: 220px;
+    font-size: 16px;
+    background-color: #E51937;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/chendler_tesla/pullRequest?prid=5700328)